### DISCUSSION TITLE
Are EXIF data stripped off by upload to Publii?

### DISCUSSION THREAD

**Discussion by 3muchfun** [GetPublii/Publii#1548] [Help] [answered]
Hi, I am experiencing the fact, that the images I load into a post as images (not galleries) are losing its EXIF data. Is this by default, and secondly, if so, can I change that?

**Comment by internettips** [comment]
@3muchfun 

Not 100% sure, but the article at: [https://getpublii.com/blog/how-to-optimise-graphical-and-media-elements.html](url) suggests that Publii does not strip the EXIF data, so leaves that to the end user. However, there have been some later versions of Publii since the article was written, so maybe the devs can confirm: @Bob or @Tomasz Dziuda

**Reply by 3muchfun** [reply]
thanks, @internettips , I read this post already, and the tip from Tomasz to remove EXIF in order to reduce file size. 
From my experience, if I export a jpg with EXIF data from Lightroom and publish the image, then I cannot find any of my EXIF data in the published image on the web. So I am looking forward to the answer of @bobmitro or @dziudek

**Comment by bobmitro** [comment]
Publii does not remove the EXIF data from images. I have checked one specifically to confirm, and it works for me very well.

**Reply by dziudek** [reply]
@bobmitro @3muchfun - exactly, in Sharp we are using withMetadata method: https://sharp.pixelplumbing.com/api-output#withmetadata but maybe JIMP removes them from thumbnails if used. Original uploaded image should remain untouched

**Reply by 3muchfun** [reply]
Ok. So, if I write a post, within this post I upload an image with EXIF data using Publii's function, and then publish this post, you say all EXIF data remain in this image when being uploaded? Secondly, is Sharp the standard Publii uses or do I have to switch it on somewhere in the App? Thanks, Michael

**Reply by 3muchfun** [reply]
I just want to figure out why my EXIF data get lost when I upload a post with an image

**Reply by bobmitro** [reply]
Attach your images here, the original and the affected one.

**Reply by 3muchfun** [reply, marked as answer]
I usually check EXIF information with Apple Fotos. By chance I saw a couple of minutes ago, that Apple Finder shows the EXIF data of the uploaded images, but Apple Fotos don't, for whatever reason.
